tests/Unleash.Tests/ClientFactory/SyncStartupUnitTest.cs

```diff
@@ -65,17 +65,6 @@ public void ImmediateInitializationBubbleAsyncErrors()
             Assert.ThrowsAsync<Exception>(async () => await unleashFactory.CreateClientAsync(settings, synchronousInitialization: true));
         }
 
-        [Test(Description = "Delayed initialization: Should only fetch toggles once")]
```

Comment:
This is now flaky. I don't actually see a way to make this work correctly, the test doesn't make a lot of sense and the interesting stuff is covered by other tests in this file